### PR TITLE
Update styling of diffs to improve a11y

### PR DESCRIPTION
**Pull request by frontdevde**
This PR includes a small visual update that fixes Issue #18551:
`Post Revisions: Improve Diff Styling for Accessibility` 

**How to test:**
Navigate to a post that has revisions and view a post revision.

**Before:**

screen shot 2017-10-10 at 12 08 22

**After:**

screen shot 2017-10-10 at 11 21 57

screen shot 2017-10-10 at 11 58 20

### CODE REVIEW

**Comment by frontdevde** [comment]
Added @shaunandrews for design review as the styling deviates from the initial design listed in https://github.com/Automattic/wp-calypso/issues/18551. See the issue ticket for discussion there.

**Comment by shaunandrews** [comment]
I dig it. 👍

**Comment by marekhrabe** [comment]
One more thing, Shaun! I don't like those hardcoded values for backgrounds. Isn't there a way we can use some kind of relative color to the base one, which uses a variable? For example lowering the opacity of the base color to 10% or something like that.

File: client/post-editor/editor-diff-viewer/style.scss [hunk 1]
```diff
@@ -22,10 +22,12 @@
 }
 
 .editor-diff-viewer__additions {
-	background: #e9f9fe;
+	background: #e8f9fe;
```
Comment (jblz) [inline client/post-editor/editor-diff-viewer/style.scss]: @frontdevde, what do you think about making these values relative to the variables we're using as the border?

e.g. `lighten( $blue-wordpress, 57% )` is fairly close to the `#e8f9fe` value here.

Comment (frontdevde) [reply client/post-editor/editor-diff-viewer/style.scss]: Happy to do it that way if it's preferred! :)

Personally, I'm not a fan of using `lighten` directly. It opens the door for a lot of different color variations throughout the code base. Ideally, I like the idea of defining even the `lighten`/`darken` shades of the colors in the colors.scss. This way we would see in one place which variations are already used and where we can consolidate. 

But that's a bigger discussion for another day. Having some kind of variable use here is definitely better than using a fixed color value. 👍

**Comment by jblz** [comment]
Thanks for switching to the relative values!

File: client/post-editor/editor-diff-viewer/style.scss (hunk 1 above)
Comment (jblz) [reply client/post-editor/editor-diff-viewer/style.scss]: >  like the idea of defining even the lighten/darken shades of the colors in the colors.scss.

Agreed. Would be happy to see these codified as such in a follow-up PR.

**Pull request merged**